Sources/Chat/ProtocolServices/Invitee/InvitationHandlingService.swift

```diff
@@ -122,7 +122,7 @@ private extension InvitationHandlingService {
                     let inviterAccount = try await identityClient.resolveIdentity(iss: claims.iss)
                     // TODO: Should we cache it?
                     let inviteePublicKey = try await identityClient.resolveInvite(account: inviterAccount)
-                    let inviterPublicKey = invite.inviterPublicKey.did(prefix: false, variant: .X25519)
+                    let inviterPublicKey = invite.inviterPublicKey.multibase(variant: .ED25519)
```

Comment:
why you changed for `ED25519`? inviterPublic key is creating by `createX25519KeyPair`

Reply:
was a typo, added did prefix too